version.gradle

```diff
@@ -1,34 +0,0 @@
-ext {
```

Comment:
This file got deleted in the feature branch. Also the build.gradle files from the affected samples in the PR.
@ruiqima